src/monkey_patch/monkey.py

```diff
@@ -68,10 +68,9 @@ class Monkey:
     logging.addLevelName(PATCH_LEVEL_NUM, "PATCH")
     logging.basicConfig(level=ALIGN_LEVEL_NUM)
     logger = logger_factory(__name__)
-    language_modeler = LanguageModel()
+    language_modeler = LanguageModel(api_model=os.getenv('API_MODEL'))
```

Comment:
We should add the `API_MODEL` envvar to `.example.env` to make it obvious that it is required.